test/built-ins/Atomics/wait/datablock-bufferdata-throws.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018 Amal Hussein. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-map-objects
+description: A DataBlock value for bufferData throws a TypeError
+info: |
+  Atomics.wait( typedArray, index, value, timeout )
+
+  1.Let buffer be ? ValidateSharedIntegerTypedArray(typedArray, true).
+    ...
+      9.If IsSharedArrayBuffer(buffer) is false, throw a TypeError exception.
+        ...
+          4.If bufferData is a Data Block, return false.
+features: [ TODO ]
```

Comment:
you can use [`Atomics`](https://github.com/tc39/test262/blob/master/features.txt#L55) here